### ISSUE TITLE
_pageBuilder bug

### ISSUE THREAD

**Issue by SeongWoo-97** [rbcprolabs/packages.flutter#180]
Hello, I'm using the native_pdf_view package well. However, we updated this package together while updating to Flutter 2.0.

After the update, I think I found a bug in pageBuilder.
I may be wrong, but please check.

```
import 'package:flutter/material.dart';
import 'package:native_pdf_view/native_pdf_view.dart';

class ViewPage extends StatefulWidget {
  final savePath;

  ViewPage({this.savePath});

  @override
  _ViewPageState createState() => _ViewPageState();
}

class _ViewPageState extends State<ViewPage> {
  PdfController pdfController;
  int _actualPageNumber = 1,
      _allPagesCount = 0;
  int previousPage = 1;

  @override
  void initState() {
    super.initState();
    pdfController = PdfController(document: PdfDocument.openFile(widget.savePath));
  }

  @override
  void dispose() {
    pdfController.dispose();
    super.dispose();
  }

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text('PDF Viewer'),
        centerTitle: true,
      ),
      body: PdfView(
          controller: pdfController,
          pageBuilder: (Future<PdfPageImage> pageImage, int index, PdfDocument document) {

            // PhotoViewGalleryPageOptions error

            return PhotoViewGalleryPageOptions(
              imageProvider: PdfPageImageProvider(
                pageImage,
                index,
                document.id,
              ),
              minScale: PhotoViewComputedScale.contained * 1,
              maxScale: PhotoViewComputedScale.contained * 3.0,
              initialScale: PhotoViewComputedScale.contained * 1.0,
              heroAttributes: PhotoViewHeroAttributes(tag: '${document.id}-$index'),
            );
          }
      ),
    );
  }
}

```

**Comment by futurelucas4502** [comment]
I have the same issue however i have been able to get a little further by adding all of these imports it functions correctly:
```dart
import 'package:native_pdf_view/native_pdf_view.dart';
import 'package:native_pdf_view/src/pdf_page_image_provider.dart';
import 'package:photo_view/photo_view_gallery.dart';
import 'package:photo_view/photo_view.dart';
```

While this is obviously not the intended way as clearly shown by it giving a warning until the bug is fixed this appears to be a workaround :)

**Comment by SeongWoo-97** [comment]
Thank you futurelucas4502.
I think I'm still lacking a lot.

**Comment by futurelucas4502** [comment]
It's alright I'd suggest keeping this issue open tho so that people know it's an issue and so it can be fixed

**Comment by SergeShkurko** [comment]
will be fixed in new release (4.1.0)

**Comment by SergeShkurko** [comment]
[native_pdf_view] v4.1.0 with Windows support published

https://pub.dev/packages/native_pdf_view/versions/4.1.0
